src/layout-grid/index.d.ts

```diff
@@ -31,6 +31,7 @@ export interface StyledGridProps {
   $gridGutters?: Responsive<number>;
   $gridMargins?: Responsive<number>;
   $gridMaxWidth?: number;
+  $gridUnit?: string;
```

Comment:
`gridUnit` should be added to `GridProps` as well.

Reply:
Also should prob add CSS types like we did with Flow definitions

Reply:
done 👍